docs/design.md

```diff
@@ -7,20 +7,15 @@ It gives an overview of the major components and interfaces, and
 provides pointers into the code. It is intended for people who wish
 to modify, fix, or extend Crest.
 
-The Homer and Homestead components are built on top of Crest. The design
-of these are discussed in the following documents, which build on the
-information described here.
-
-* [Homer Design](homer_design.md)
-* [Homestead Design](homestead_design.md)
+The Homer and Homestead-prov components are built on top of Crest. 
 
 Requirement
 ===========
 
-Crest is used in Homer and Homestead, two components which act as
+Crest is used in Homer and Homestead-prov, two components which act as
 distributed databases in a Clearwater system. Homer is an XDMS server,
-which stores the call services for subscribers, while Homestead is
-a HSS cache, storing authentication and iFCs.
+which stores the call services for subscribers, while Homestead-Prov is
+a provisioning interface to Homestead, our [HSS mirror](https://github.com/Metaswitch/homestead).
```

Comment:
How about 'a provisioning interface to [Homestead](https://github.com/Metaswitch/homestead), which changes it from being a HSS cache into a standalone subscriber store.'?